wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Annual: update to new design

Fixes #11907 

This renames Insights _Annual Site Stats_ to _This Year_, and updates it to use the two-column stat card.

To test:

- On a site with no posts, go to Insights _This Year_.
  - Verify `No data yet` is shown.
- On a site with posts, go to Insights _This Year_.
  - Verify the data is shown in two columns.
  - Verify `View more` is shown. To note, it does nothing right now. That will be added with #11908.

Update release notes:


---
<img width="400" alt="this_year_no_data" src="https://user-images.githubusercontent.com/1816888/59642282-96b67900-9121-11e9-9f6d-008a6288df28.png">

---
<img width="400" alt="this_year" src="https://user-images.githubusercontent.com/1816888/59642297-9ddd8700-9121-11e9-8e5d-eca88bbc3539.png">
